packages/terra-core-docs/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Changed
+  * Move terra-header docs out of example directory
```

Comment:
nit: "out of" is vague and does not say where they go at a glance
```suggestion
  * Moved terra-header docs to root heading directory.
```

Reply:
@sdadn Good point. Though "root heading directory" could also mean `packages/terra-heading`
And I accidentally called `terra-heading` `terra-header`

How's
```suggestion
  * Moved terra-heading docs to root terra-header docs directory
```
?

Reply:
You're right about the confusion. Let's just give the path for simplicity then:

```suggestion
  * Moved terra-heading docs up one level to `src/terra-dev-site/doc/heading`
```